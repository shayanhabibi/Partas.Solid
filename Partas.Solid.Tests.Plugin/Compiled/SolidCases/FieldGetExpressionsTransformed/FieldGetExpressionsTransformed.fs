module Partas.Solid.Tests.SolidCases.FieldGetExpressionsTransformed

open Partas.Solid
open Fable.Core.JS
open Fable.Core
open Fable.Core.JsInterop

[<Erase>]
type WordRotate() =
    inherit div()
    [<Erase>] member val words: string[] = unbox null with get,set
    [<Erase>] member val duration: int = unbox null with get,set
    [<SolidTypeComponentAttribute>]
    member props.constructor =
        let index,setIndex = createSignal(0)
        let chog = props.words.Length
        div()

