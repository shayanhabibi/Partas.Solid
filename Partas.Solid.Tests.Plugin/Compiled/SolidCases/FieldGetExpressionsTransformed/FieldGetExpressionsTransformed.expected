import { createSignal, splitProps } from "solid-js";

export function WordRotate(props) {
    const [PARTAS_LOCAL, PARTAS_OTHERS] = splitProps(props, ["words"]);
    const patternInput = createSignal(0);
    const chog = PARTAS_LOCAL.words.length | 0;
    return <div />;
}

